Tools/build_standalone.py

```diff
@@ -0,0 +1,102 @@
+"""Builds a standalone player for the current operating system.
+
+The build output is written to the <build root>/<timestamp> directory.
+"""
+
+import datetime
+import os
+import platform
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+
+import unity
+from absl import app, flags, logging
+
+FLAGS = flags.FLAGS
+
+# Path to the repository root.
+REPO_ROOT = Path(__file__).resolve().parents[1]
+
+# Path to the default build root directory.
+DEFAULT_BUILD_ROOT = REPO_ROOT / "Build"
+
+# Default name of the standalone build.
+DEFAULT_BUILD_NAME = "micromissiles"
+
+
+@dataclass(frozen=True)
+class PlatformTarget:
+    """Platform target."""
+    unity_name: str
+    build_flag: str
+    extension: str
```

Comment:
add detailed docstring
```suggestion
@dataclass(frozen=True)
class PlatformTarget:
    """Platform-specific build target configuration.

    Attributes:
        unity_name: Unity's internal platform name (e.g., "StandaloneWindows64").
        build_flag: Command-line flag for building this platform.
        extension: File extension for the built executable (empty string for Linux).
    """
```